jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -46,7 +46,9 @@
 	private static final String FORMAT_PREFIX = " Format: "; //$NON-NLS-1$
 
 	@SuppressWarnings("nls")
-	private static final String FILE_PATH_HELP = " <file path>%n"
+	private static final String DUMP_OPTION_HELP = " [request=<options>] [opts=<options] [<file path>]%n"
```

Comment:
does the order of these three entries matter? can we turn the `<file path>` to the similar key=value style, i.e., `file=<file path>` (`dsn=<file path>` for `zOS`)?

Reply:
Seems like that would be a breaking change. Some users may already be using `jcmd $pid Dump.system $file`, for example.

Reply:
@kgibm Agreed that it will require a script change for the usage like your example. On the other hand, my feeling is that the proposed option format are not consistent, and might cause confusion down the road, IMO, maybe now it is the time to provide unified option format matching `-Xdump`.
Can the example usage be modified to accommodate a new format hopefully with low cost?

Reply:
@JasonFengJ9 That's a fair point. I defer to others to decide. I guess one middle ground could be to support both: add support for `file=`/`dns=` and then assume that if an argument is not preceded with `word=`, then it is the file for backwards compatibility.

Reply:
The order doesn't matter. We could allow `file=` and `dsn`, but I don't want to break compatibility by breaking the existing syntax.

Reply:
Sure, no objection if the back compatibility takes precedence. 

Also got another question, in current option format, just wondering how do we address `The command can be extended to take other parameters as well so the user can set any options` [1] (assuming such options available in `-Xdump`), or in a separated PR instead?

[1] https://github.com/eclipse/openj9/issues/10747#issue-711503883

Reply:
This was already done by accepting `request=` and `opts=`. Besides `file=` or `dsn-`, these are the only valid options when requesting an immediate dump.

Reply:
Okay, my thought was to specify same style of `Xdump` options via `jcmd`, i.e., dump accordingly as per the `events`, `filter`, and others. I am fine with the interpretation of an immediate dump for this PR.

Reply:
Changing the existing dump options is a different operation, which could also be added, but separately from this change.

Reply:
I don't think supporting `dsn=` or `file=` makes sense. The API of `com.sun.tools.attach.spi.AttachProvider` doesn't promise that the virtual machines answered are on the same host (as each other or as where `jcmd` is used). As such, it's unclear whether `dsn=` or `file=` would be appropriate paired with a particular vmid. I think the decision about how to choose the appropriate option string is best left to the target vm.

Reply:
Technically a target vm could state its OS type (`zOS` or not) via `VMID` in its advertisement file.
On the other hand, `zOS` is a tightly controlled system which even disables the `AttachAPI` by default, it will be rare situation that there is an interaction between `zOS` images and other OS.
The motivation to add `dsn=` or `file=` is to reduce the parsing code complexity and the whole string can be consumed by native code in a whole.